android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -113,6 +113,7 @@ public void initNativeSdk(final ReadableMap rnOptions, Promise promise) {
             }
 
             options.setSentryClientName(sdkVersion.getName() + "/" + sdkVersion.getVersion());
+            options.setNativeSdkName("sentry.native.android.react-native");
```

Comment:
Let's use the already existing variable for the name:
https://github.com/getsentry/sentry-react-native/blob/2d064085c1bb8e07ee516ecc6711232f42fbc5cc/android/src/main/java/io/sentry/react/RNSentryModuleImpl.java#L71

It's later used for assigning correct origin and environment tags:
https://github.com/getsentry/sentry-react-native/blob/2d064085c1bb8e07ee516ecc6711232f42fbc5cc/android/src/main/java/io/sentry/react/RNSentryModuleImpl.java#L624-L626